src/Model/APContact.php

```diff
@@ -241,9 +241,11 @@ public static function getByURL($url, $update = null)
 			$apcontact['name'] = $apcontact['nick'];
 		}
 
-		$apcontact['about'] = HTML::toBBCode(JsonLD::fetchElement($compacted, 'as:summary', '@value'));
+		$aboutHtml = JsonLD::fetchElement($compacted, 'as:summary', '@value');
+		$apcontact['about'] = (!empty($aboutHtml) ? HTML::toBBCode($aboutHtml) : '');
```

Comment:
```suggestion
		$apcontact['about'] = HTML::toBBCode(JsonLD::fetchElement($compacted, 'as:summary', '@value') ?? '');
```

Reply:
Done and thank you for your patience. :-)